Makefile

```diff
@@ -12,16 +13,47 @@ LDFLAGS += -flat_namespace -undefined suppress -dynamiclib
 SO = dylib
 else 
 LDFLAGS += -shared
-SO = so
+SHLIB_MAJOR:=1
+SHLIB_MINOR:=0
+SHLIB_PATCHLEVEL:=0
```

Comment:
why 1.0.0? we don't really guarantee any compatibility, so it'd probably be more semver-like to keep it 0.x.y?

Reply:
see email response below